app/controllers/cases/offender_sar_controller.rb

```diff
@@ -210,6 +212,27 @@ def confirm_sent_to_sscl
       redirect_to case_path(@case) and return
     end
 
+    def information_received
```

Comment:
This method isn't explicitly needed if this is all it is doing. It should render without the method.